auth/kerberos/src/krb.cpp

```diff
@@ -798,51 +798,37 @@ std::list<std::string> renew_kerberos_tickets_domainless(std::string krb_files_d
         {
             std::pair<int, std::string> gmsa_ticket_result;
             std::string krb_cc_name = krb_ticket->krb_file_path;
-            // check if the ticket is ready for renewal
-            if ( is_ticket_ready_for_renewal( krb_cc_name ) )
+            int num_retries = 1;
```

Comment:
Please add a comment on reason for retries

Reply:
added comment about retries